tests/queries/server.py

```diff
@@ -137,19 +137,34 @@ def stop(self):
         </test_shard_localhost>
 
         <test_cluster_two_shards_localhost>
-             <shard>
-                 <replica>
-                     <host>localhost</host>
-                     <port>{tcp_port}</port>
-                 </replica>
-             </shard>
-             <shard>
-                 <replica>
-                     <host>localhost</host>
-                     <port>{tcp_port}</port>
-                 </replica>
-             </shard>
-         </test_cluster_two_shards_localhost>
+            <shard>
```

Comment:
What is the purpose of this change?

Reply:
To allow more tests to pass - using pytest framework. Without this section of config they don't work.